pkgs/top-level/all-packages.nix

```
@@ -36175,6 +36177,7 @@ with pkgs;
   };
   transmission-gtk = transmission.override { enableGTK3 = true; };
   transmission-qt = transmission.override { enableQt = true; };
+  transmission-withoutSystemd = transmission.override { enableSystemd = false; };
```

```suggestion
  transmission_noSystemd = transmission.override { enableSystemd = false; };
```

I think we're meant to use underscores for variants (as opposed to `-` for actual names of packages).
[naming guidelines](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)